.travis.yml

```diff
@@ -28,14 +28,14 @@ install:
     - npm install -g jslint
 
       # Download a Selenium Web Driver release
-    - wget "http://selenium-release.storage.googleapis.com/2.45/selenium-server-standalone-2.45.0.jar"
+    - wget "http://selenium-release.storage.googleapis.com/2.52/selenium-server-standalone-2.52.0.jar"
 
     - PHP_INI_SCAN_DIR=":test/" php -S localhost:8000 -t htdocs htdocs/router.php 2>1 > /dev/null & 
 
     # Start Selenium and redirect Selenium WebDriver
     # output to /dev/null so that it doesn't flood the
     # screen in the middle of our other tests
-    - DISPLAY=:99.0 java -jar selenium-server-standalone-2.45.0.jar  2>1 > /dev/null &
+    - DISPLAY=:99.0 java -jar selenium-server-standalone-2.52.0.jar  2>1 > /dev/null &
```

Comment:
Got it. As  @driusan  said, it might just be related to some delay on Travis server. I re-sent the pull request and it is now queued